src/include/type/transient_value.h

```diff
@@ -0,0 +1,346 @@
+#pragma once
```

Comment:
Wait, we got rid of transient value. Can you check how you merged this?

Reply:
I originally had two branches - the old one with all the large json changes and a up to date branch with some minor changes to get the json library to work correctly. The way I made this branch is I merged the first branch into an upto date master branch and then merged the second branch into it as well. 

I realize now this was not a good way to do things. I'm thinking the cleanest thing is to merge master into the old json branch with all the large changes and submit a new pull request. 

My bad, let me know what you suggest/think is best.